tests/npm-app/app.json

```diff
@@ -1,203 +1,272 @@
+                        "type": "number"
+                      },
+                      "msg": {
+                        "type": "string"
+                      }
+                    },
+                    "type": "object"
                   }
                 }
               }
             }
           }
         }
       },
-      "/proto": {
-        "post": {
-          "js_module": "src/endpoints/proto.js",
-          "js_function": "proto",
-          "forwarding_required": "always",
-          "execute_locally": false,
-          "require_client_signature": false,
-          "require_client_identity": true,
-          "readonly": false,
-          "openapi": {
-            "requestBody": {
-              "required": true,
+      "post": {
+        "js_module": "src/endpoints/log.js",
+        "js_function": "setLogItem",
+        "forwarding_required": "always",
+        "execute_locally": false,
+        "require_client_signature": false,
+        "require_client_identity": true,
+        "readonly": false,
+        "openapi": {
+          "responses": {
+            "200": {
+              "description": "Ok",
               "content": {
-                "text/plain": {}
+                "application/json": {
+                  "schema": {}
+                }
               }
-            },
-            "responses": {
-              "200": {
-                "content": {
-                  "application/x-protobuf": {}
+            }
+          },
+          "requestBody": {
+            "required": true,
+            "content": {
+              "application/json": {
+                "schema": {
+                  "properties": {
+                    "msg": {
+                      "type": "string"
+                    }
+                  },
+                  "type": "object"
                 }
               }
             }
           }
         }
-      },
-      "/log": {
-        "get": {
-          "js_module": "src/endpoints/log.js",
-          "js_function": "getLogItem",
-          "forwarding_required": "always",
-          "execute_locally": false,
-          "require_client_signature": false,
-          "require_client_identity": true,
-          "readonly": false,
-          "openapi": {
-            "responses": {
-              "200": {
-                "description": "Ok",
-                "content": {
-                  "application/json": {
-                    "schema": {
+      }
+    },
+    "/log/all": {
+      "get": {
+        "js_module": "src/endpoints/log.js",
+        "js_function": "getAllLogItems",
+        "forwarding_required": "always",
+        "execute_locally": false,
+        "require_client_signature": false,
+        "require_client_identity": true,
+        "readonly": false,
+        "openapi": {
+          "responses": {
+            "200": {
+              "description": "Ok",
+              "content": {
+                "application/json": {
+                  "schema": {
```

Comment:
Does seem to match the code?

Reply:
Good spot - intended to make this an object but used a tuple in the code. Now an object, so schema and code should match.